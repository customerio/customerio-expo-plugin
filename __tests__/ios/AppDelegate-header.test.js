const fs = require("fs-extra");
const path = require("path");

const testProjectPath = path.join(__dirname, "../../test-app");
const iosPath = path.join(testProjectPath, "ios");
const appDelegateHeaderPath = path.join(iosPath, "ExpoTestApp/AppDelegate.h");

test("Plugin injects CIO imports and calls into AppDelegate.h", async () => {
  const content = await fs.readFile(appDelegateHeaderPath, "utf8");

  expect(content).toMatchSnapshot();
});
