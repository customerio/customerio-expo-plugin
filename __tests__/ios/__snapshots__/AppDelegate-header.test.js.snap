// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugin injects CIO imports and calls into AppDelegate.h 1`] = `
"#import <RCTAppDelegate.h>
#import <UIKit/UIKit.h>
#import <Expo/Expo.h>

#import <UserNotifications/UserNotifications.h>
@interface AppDelegate : EXAppDelegateWrapper <UNUserNotificationCenterDelegate>
@end
"
`;
